llvm/lib/Target/SPIRV/SPIRVPrepareGlobals.cpp

```diff
@@ -43,6 +44,29 @@ bool tryExtendLLVMBitcodeMarker(GlobalVariable &Bitcode) {
   return true;
 }
 
+bool tryExtendDynamicLDSGlobal(GlobalVariable &GV) {
+  constexpr unsigned WorkgroupAS = 3;
```

Comment:
It is very unlinkely that this will change, but could this be changed to used named constants:

```suggestion
  constexpr unsigned WorkgroupAS = storageClassToAddressSpace(SPIRV::StorageClass::Workgroup);
```

It should all get folded at compile time since it is all constexpr.